src/black/__init__.py

```diff
@@ -273,9 +273,9 @@ class Mode:
     target_versions: Set[TargetVersion] = field(default_factory=set)
     line_length: int = DEFAULT_LINE_LENGTH
     string_normalization: bool = True
+    is_pyi: bool = False
```

Comment:
Made them the same order as they appear in the cache file name for ease of understanding.